CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changes to Calva.
 
 ## [Unreleased]
 
+## [2.0.391] - 2023-10-11
+
+- Fix: [Allowing dram download to fail vocally for user. Fixing a JS join path error](https://github.com/BetterThanTomorrow/calva/issues/2325)
```

Comment:
I was reviewing a bit too fast. This entry should not have a date, nor a version. It should remain just **Unreleased**. But no worries, I fixed it on the `dev` branch.